client/sites/staging-site/components/staging-site-transfer-banner/index.tsx

```diff
@@ -0,0 +1,66 @@
+import { Gridicon } from '@automattic/components';
+import {
+	__experimentalText as Text,
+	__experimentalHeading as Heading,
+	__experimentalVStack as VStack,
+	__experimentalHStack as HStack,
+	Card,
+	CardBody,
+} from '@wordpress/components';
+import { __ } from '@wordpress/i18n';
+import stagingSiteTransferBackground from './staging-site-transfer-banner-background.svg';
+import stagingSiteTransferIllustration from './staging-site-transfer-banner-illustration.svg';
+
+export default function StagingSiteTransferBanner() {
+	return (
+		<Card>
+			<CardBody
+				style={ {
+					padding: '40px',
+					background: `
+							linear-gradient(
+								to right,
+								rgba( 255, 255, 255, 1 ) 0%,
+								rgba( 255, 255, 255, 0.41 ) 59%,
+								rgba( 255, 255, 255, 0 ) 100%
+							),
+							url(${ stagingSiteTransferBackground }) repeat
+						`,
+				} }
+			>
+				<HStack spacing={ 4 }>
+					<VStack>
+						<Heading level={ 1 } weight={ 500 }>
+							{ __( 'Creating staging site' ) }
+						</Heading>
+						<Text as="p" variant="muted">
+							{ __(
+								'We’re currently creating your staging site. This may take a few moments, depending on the size of your site.'
```

Comment:
One thought that came to my mind is that the banner could let the user know that they will receive an email once the staging site is created. Something like:

<img width="4540" height="1768" alt="Markup on 2025-08-05 at 10:53:31" src="https://github.com/user-attachments/assets/76198351-29d3-4383-85d2-d650ec0b183f" />


```suggestion
								'We’re currently creating your staging site. This may take a few moments, depending on the size of your site. You’ll receive an email once it’s ready.'
```

cc @Marinatsu  

But we can leave the things as they are now. Just an idea.

Reply:
IMO it's a good improvement, I added it and we can change it if needed.